rotkehlchen/tests/api/test_statistics.py

```diff
@@ -209,6 +209,67 @@ def test_query_statistics_asset_balance_errors(rotkehlchen_api_server, rest_api_
     )
 
 
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_statistics_asset_balance_inferred_zeros(rotkehlchen_api_server):
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    formatstr = '%Y-%m-%dT%H:%M:%SZ'
```

Comment:
can you add a small docstring explaining what is being tested here? it helps later when navigating the tests